types/angular-material/index.d.ts

```diff
@@ -462,6 +462,7 @@ declare module 'angular' {
             openFrom(from: string | Element | Event | { top: number, left: number }): IPanelAnimation;
             closeTo(to: string | Element | { top: number, left: number }): IPanelAnimation;
             withAnimation(cssClass: string | { open: string, close: string }): IPanelAnimation;
+            duration(duration: number | { open: string, close: string }): IPanelAnimation;
```

Comment:
this shourl read `number`, not `string`:
https://material.angularjs.org/latest/api/type/MdPanelAnimation#mdpanelanimation-duration-duration